src/olympia/blocklist/tests/test_cron.py

```diff
@@ -185,13 +187,19 @@ def test_waffle_off_disables_publishing(self):
         self.publish_record_mock.assert_not_called()
         assert not get_config(MLBF_TIME_CONFIG_KEY)
 
-    def test_no_block_changes(self):
+    @freeze_time('2020-01-01 12:34:56', as_arg=True)
+    def test_no_block_changes(frozen_time, self):
         # This was the last time the mlbf was generated
         last_time = int(
             datetime.datetime(2020, 1, 1, 12, 34, 1).timestamp() * 1000)
         # And the Block was modified just before so would be included
         self.block.update(modified=datetime.datetime(2020, 1, 1, 12, 34, 0))
         set_config(MLBF_TIME_CONFIG_KEY, last_time, json_value=True)
+        prev_blocked_path = os.path.join(
+            settings.MLBF_STORAGE_PATH, str(last_time), 'blocked.json')
+        with storage.open(prev_blocked_path, 'w') as blocked_file:
```

Comment:
What exactly is happening here?

Reply:
now we're using the json of previously blocked guid:version to compare to, the file needs to exist so it can be read.